packages/starter/example/pages/index.tsx

```diff
@@ -11,10 +11,11 @@ export default function Index() {
                 alignItems: "center",
             }}
         >
-            <CrossMintButton
+            <CrossmintPayButton
                 collectionTitle="Test Collection"
                 collectionDescription="This is a CrossMint test collection!"
                 collectionPhoto="https://picsum.photos/300"
+                clientId="058eeac7-3751-4c66-afc3-f77270c9d636"
```

Comment:
was it intended to add an id here?

Reply:
This is what we had in the `<CrossMintProvider>`. I just moved it to the button.